packages/purpose-process/test/purposeService.integration.test.ts

```diff
@@ -65,9 +102,92 @@ describe("database test", async () => {
   });
 
   describe("Purpose service", () => {
-    describe("create purpose", () => {
-      it("TO DO", () => {
-        expect(1).toBe(1);
+    const mockPurpose = getMockPurpose();
+    const mockEService = getMockEService();
```

Comment:
```suggestion
```
Since they just call the getMock I don't think they are useful, you can just call the function when you need it

Reply:
Updated